playground/backend/internal/errors/grpc_errors_test.go

```diff
@@ -49,21 +72,44 @@ func TestInternalError(t *testing.T) {
 
 func TestInvalidArgumentError(t *testing.T) {
 	type args struct {
-		title   string
-		message string
+		title         string
+		formatMessage string
+		arg           []interface{}
 	}
 	tests := []struct {
 		name     string
 		args     args
 		expected string
 		wantErr  bool
 	}{
-		{name: "TestInvalidArgumentError", args: args{title: "TEST_TITLE", message: "TEST_MESSAGE"},
-			expected: "rpc error: code = InvalidArgument desc = TEST_TITLE: TEST_MESSAGE", wantErr: true},
+		{
+			// Test case with calling InvalidArgumentError method.
```

Comment:
It seems like we can reduce the number of comments here and in all similar cases. I mean - the single difference in all comments - only value from the field "name". Can we add a joint statement to all cases?

Reply:
Changed comments.